templates/default/cg_foodcritic_tests.rb.erb

```diff
@@ -39,6 +39,9 @@ def dependency_versions(ast)
   deps
 end
 
+def depends_on_specific_version(version)
```

Comment:
In ruby it's custom to have a question-mark in the function name if it returns a bool.. .So this should be `def depends_on_specific_version?(version)` right?